samples/bootloader/README.rst

```diff
@@ -218,12 +218,18 @@ Firmware versions using the |NSIB| are kept in the form of a *monotonic counter*
 Counter values are kept as slots in the flash memory, with each new counter value occupying a new slot.
 See :kconfig:option:`CONFIG_SB_MONOTONIC_COUNTER` for more details.
 
-To set the counter value of an application, build it using the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option:
+You can set the counter value for different elements in the bootloader sequence.
+Using :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option in ``prj.conf`` will allow you to change the version of the application image.
+However, in order to specify :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` for the MCUboot image you have to create a ``child_image`` folder in the project directory.
+For more information regarding the child image configuration refer to :ref:`ug_multi_image_build_scripts` section.
```

Comment:
I think the link goes to the wrong section. Shouldn't it go to [Image-specific variables](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8831/nrf/ug_multi_image.html#image-specific-variables)? Ideally the paragraph that starts with «You can make a project pass Kconfig configuration files, ...» if that is possible.

Reply:
reference updated